tests/bluetooth/bsim_bt/compile.source

```diff
@@ -1,6 +1,12 @@
 # Copyright 2018 Oticon A/S
 # SPDX-License-Identifier: Apache-2.0
 
+function print_error_info(){
+  echo -e "\033[0;31mFailure building ${app} ${conf_file} for ${BOARD}\033[0m\n\
```

Comment:
Is the `\033[0;31m` and  `\033[0m` intentional?

Reply:
Yes, they are the ascii scapes for red and back to normal